### PR TITLE
[CoreBundle] User disabling

### PR DESCRIPTION
**Pull request by ngodfraind**
- Users can be disabled in the administration
- a new 'is_removed' field exists to perform soft delete operations (wich were previously handled by the is_enabled field).

References #944

### CODE REVIEW

**Comment by LaurentGruber** [comment]
LGTM

File: main/core/Repository/UserRepository.php [hunk 1]
```diff
@@ -871,7 +871,7 @@ public function findEnabledUserById($userId)
             SELECT u
             FROM Claroline\CoreBundle\Entity\User u
             WHERE u.id = :userId
-            AND u.isEnabled = TRUE
+            AND u.isRemoved = TRUE
```
Comment (stefk) [inline main/core/Repository/UserRepository.php]: Shouldn't it be false instead?

File: main/core/Repository/UserRepository.php [hunk 2]
```diff
@@ -885,7 +885,7 @@ public function findAllEnabledUsers($executeQuery = true)
         $dql = '
             SELECT u
             FROM Claroline\CoreBundle\Entity\User u
-            WHERE u.isEnabled = TRUE
+            WHERE u.isRemoved = TRUE
```
Comment (stefk) [inline main/core/Repository/UserRepository.php]: Same here.

File: main/core/Repository/UserRepository.php (hunk 1 above)
Comment (ngodfraind) [inline main/core/Repository/UserRepository.php]: Good catch. I used some a search and replace but I did the first one manually and probably messed some one thel up.

**Comment by ngodfraind** [comment]
I removed one the the methods (not used anywhere) and fixed the other one. It was used in a 4.xx  updater and in a debug command so it's pretty minor (and it probably could be removed aswell with some refactoring).

**Comment by stefk** [comment]
LGTM

**Pull request merged**